docs/_posts/0000-05-01-oo-inheritance.md

```diff
@@ -131,24 +110,30 @@ Work in language groups to collaboratively come up with a diagram
 
 ## Exercise Review and Implementation
 
-```text
+<mermaid>
+classDiagram
+    Card <|-- AnimalCard : implements
+    Card <|-- PlayingCard : implements
```

Comment:
Probably a copy paste error - this should be inheritance:
```suggestion
    Card <|-- AnimalCard
    Card <|-- PlayingCard
```

Reply:
Thanks! I’ll change these to `extends` rather than `implements`.